src/sentry/api/serializers/models/release.py

```diff
@@ -45,13 +45,14 @@ def get_users_for_commits(item_list):
         id__in=[ue.user_id for ue in user_emails],
         sentry_orgmember_set__organization_id=org_id
     )
-    users_by_id = dict((user.id, serialize(user)) for user in users)
+    users = serialize(list(users))
+    users_by_id = dict((user['id'], user) for user in users)
```

Comment:
you could also do this as a dictionary comprehension: `users_by_id = {user['id']: user for user in users}`